package.json

```diff
@@ -33,10 +33,11 @@
   "dependencies": {
     "ethers": "^5.6.5",
     "tap-spec": "^5.0.0",
-    "tape": "^5.5.3"
+    "tape": "^5.5.3",
+    "viem": "^2.21.1"
```

Comment:
can we make this a peer dependency so that it doesn't get included in people's projects unsolicited?

or idk, make sure that just the types are included

Reply:
Including only the type is not possible since there is no typeDef package for viem.
Have added it as a peer dependency